keyrings/live/blackbox-admins.txt

```diff
@@ -0,0 +1,2 @@
+beamalsky@gmail.com
+reginafcompton@gmail.com
```

Comment:
Either Derek or Forest should be on every project. Let's add Derek.

Reply:
Added!